lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -498,6 +498,12 @@ sub overview {
     }
 
     if (!$search_args{build}) {
+        if (@groups > 1) {
+            return $self->render(text => 'Specify a build when you want to lookup multiple groups', status => 404);
+        }
+        elsif (@groups == 1) {
+            $search_args{groupid} = $groups[0]->id;
```

Comment:
This line is not covered - which would be the actual change. So please add a test case